lambdas/mail_save.js

```diff
@@ -32,13 +32,13 @@ const handler = async(req, res) => {
           if (err) {
             send(res, statuses['bad request'], err.message);
```

Comment:
This error comes back from mailgun in english, we might want to use a generic spanish text; the only error I have seen so far is when your email is already subscribed, so it could go something lik 'Este email ya esta registrado'.